antelope_contracts/tests/erc20/contracts.hpp.in

```diff
@@ -32,6 +32,8 @@ struct contracts {
 
     static std::vector<uint8_t> evm_stub_wasm() { return read_wasm("${ANTELOPE_CONTRACTS_BINARY_DIR}/stubs/stub_evm_runtime.wasm"); }
     static std::vector<char> evm_stub_abi() { return read_abi("${ANTELOPE_CONTRACTS_BINARY_DIR}/stubs/stub_evm_runtime.abi"); }
+
+    static std::vector<uint8_t> evm_depoit_proxy_wasm() { return read_wasm("${ANTELOPE_CONTRACTS_BINARY_DIR}/deposit_proxy/deposit_proxy.wasm"); }
```

Comment:
Typo. Should be `evm_deposit_proxy_wasm()`.

I know there are no actions right now, but assuming it still generates an ABI, it would be good to continue following the pattern of deploying the ABI alongside the WASM anyway.